src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -454,6 +472,54 @@ private void LogAndValidateOptions(MigrateRepoCommandArgs args)
                 _log.LogInformation($"GIT ARCHIVE URL: {args.GitArchiveUrl}");
                 _log.LogInformation($"METADATA ARCHIVE URL: {args.MetadataArchiveUrl}");
             }
+            if (args.LfsMappingFile != "")
+            {
+                _log.LogInformation($"LFS MAPPING FILE: {args.LfsMappingFile}");
+            }
+        }
+
+        private byte[] applyLfsMappingFileToMetadata(byte[] metadataArchiveContent, string lfsMappingFile)
+        {
+            // extract the metadata archive
+            var gzipDataStream = new MemoryStream(metadataArchiveContent);
+            Stream inStream = new GZipInputStream(gzipDataStream);
+            TarArchive archive = TarArchive.CreateInputTarArchive(inStream, TarBuffer.DefaultBlockFactor);
```

Comment:
1. All of these classes are `IDisposable` so they should be properly disposed. In C# you can do it with `using`: 
```c#
using var gZipData = new MemoryStream(metadataArchiveContent);
```
2. Prefer `var variable` over `Type varialbe` --> `var inStream = ...`
3. Generally speaking we should avoid Hungarian notation --> use `gZipData`  instead of `gZipDataStream`.

We follow these guidelines for naming conventions (with some exceptions, for example we use all caps for constants): 
1. [Microsoft C# naming guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines)
2. [The most widely used guideline (non Microsoft)](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md)

Reply:
Some of this is Greek (or Hungarian 😂 ) to me.